packages/realm/package.json

```diff
@@ -185,16 +187,18 @@
       "command": "node ../../scripts/build-android.js",
       "files": [
         "../../src/android/**",
-        "../bindgen/src/**/*.ts",
-        "../bindgen/src/**/*.h"
+        "bindgen/src/**/*.ts",
+        "bindgen/vendor/bindgen-lib/src/**/*.ts",
+        "bindgen/vendor/bindgen-lib/src/**/*.h"
```

Comment:
Will there not be any `.h` files under `"bindgen/src/**/"`?

Reply:
There will be, but I haven't moved them yet. I'll do that soon.